ui/admin/app/controllers/scopes/scope/session-recordings/index.js

```diff
@@ -95,29 +106,95 @@ export default class ScopesScopeSessionRecordingsIndexController extends Control
   }
 
   /**
-   * Returns unique project scopes from targets
-   * linked to the session recordings
+   * Configurations for different filter options
    * @returns {[object]}
    */
-  get projectScopes() {
-    const uniqueMap = new Map();
-    this.model.allSessionRecordings.forEach(
-      ({
-        create_time_values: {
-          target: {
-            scope: { id, name, parent_scope_id },
-          },
+  get filterConfigs() {
+    return {
+      userFilters: {
+        select: [
+          { field: 'user_id', isDistinct: true },
+          { field: 'user_name', isDistinct: true },
```

Comment:
would it complicate the select objects too much if it were changed so that `isDistinct` is only specified once since it looks like the parser only looks to see if at least one field isDistinct? Totally get if the object is structured this way for consistency.

Reply:
So the original reason was I didn't want to complicate it to take in both an array and an object for the `select` and was planning on letting users mix `isDistinct` with other options. When testing, I realized there's not a current use case to mix them so I kept it simple by just assuming you'd only be using distinct clauses if it's there but there's no reason it has to be like that in the future. I'd probably keep it like this just to make the API simpler and for future compatibility and not assume that the presence of any `isDistinct` means all of them are